wp-pgp-encrypted-emails.php

```diff
@@ -1273,14 +1225,14 @@ private static function prepareMail ($to, $subject, $message, $headers, $attachm
             }
         }
         else if ($pub_cert) {
+            $clearfile = tempnam("temp","email");
```

Comment:
This will not actually create a temporary file in the `temp` directory because that directory doesn't exist, but rather in the system's default temporary directory, so it might be worth being more explicit here (i.e., with `tempnam( '/tmp', 'email' );`) or, better yet, making sure this is written to a *specific* writable but not Web-accessible directory as a defense-in-depth approach, such as creating a directory with a random name in the WordPress writable directory, or just use that directory explicitly ([`wp_get_upload_dir()`](https://developer.wordpress.org/reference/functions/wp_get_upload_dir/)) instead of `/tmp` so that you are certain you know where the code will write the file.